### PR TITLE
3 login page

### PR DESCRIPTION
**Pull request by devhyun05**
Verify all information before logging in - working
Display error massage when information wrong - working
Redirect to MainPage - working

### CODE REVIEW

File: CollegeApplication/Screens/LoginView.swift [hunk 1]
```diff
@@ -45,39 +46,68 @@ struct LoginView: View {
...
+            self.errorMessage = "Empty Username"
+            return
+        } else if password.isEmpty {
+            wrongPassword = 2
+            self.errorMessage = "Empty Password"
+            return
+        }
+        
         guard let user = users.first(where: {$0.username == username.lowercased()}) else{
             wrongUsername = 2
```
Comment (renhotsai) [inline CollegeApplication/Screens/LoginView.swift:87]: is any possible change to enum?

Comment (devhyun05) [reply CollegeApplication/Screens/LoginView.swift:87]: That's a good idea. Let me change

**Review by devhyun05 — COMMENTED** [review]
How about now? (Localized for multi-language support for future)

File: CollegeApplication/Screens/LoginView.swift [hunk 2]
```diff
@@ -75,23 +75,23 @@ struct LoginView: View {
         
         if username.isEmpty {
             wrongUsername = 2
```
Comment (renhotsai) [inline CollegeApplication/Screens/LoginView.swift:77]: what's "2" for? If it just for error border, maybe use Bool to control  will be better.

Comment (devhyun05) [reply CollegeApplication/Screens/LoginView.swift:77]: Sure, working on it

**Pull request merged**